### PR TITLE
feat(web): GRW-12 - create a bridge app with sample workflows for new signups

### PR DESCRIPTION
**Pull request by scopsy**
- Syncs new signups to a sample bridge endpoint with 5 workflows
- Shows a top level banner on all workflows page indicating that you are connected to a sample bridge endpoint with CTA to complete onboarding
- minor copy updates

### CODE REVIEW

File: apps/api/src/app/bridge/usecases/sync/sync.usecase.ts [hunk 1]
```diff
@@ -57,13 +57,15 @@ export class Sync {
...
     }
 
-    this.analyticsService.track('Sync Request - [Bridge API]', command.userId, {
-      _organization: command.organizationId,
-      _environment: command.environmentId,
-      workflowsCount: discover.workflows?.length || 0,
-      localEnvironment: !!command.bridgeUrl?.includes('novu.sh') ? true : undefined,
-      source: command.source,
-    });
+    if (command.source !== 'sample-workspace') {
```
Comment (scopsy) [inline apps/api/src/app/bridge/usecases/sync/sync.usecase.ts:60]: Don't fire the event for sample sync

File: apps/web/src/AppRoutes.tsx [hunk 2]
```diff
@@ -119,8 +120,8 @@ export const AppRoutes = () => {
           <Route path="create" element={<CreateTenantPage />} />
           <Route path=":identifier" element={<UpdateTenantPage />} />
         </Route>
-        {isImprovedOnboardingEnabled ? (
-          <Route path={ROUTES.GET_STARTED} element={<GetStartedPage />} />
+        {isV2Enabled ? (
+          <Route path={ROUTES.GET_STARTED} element={<GetStartedPageV2 location="get-started" />} />
```
Comment (scopsy) [inline apps/web/src/AppRoutes.tsx:124]: Restored the Getting Started Page

File: apps/web/src/components/layout/components/PrivatePageLayout.tsx [hunk 3]
```diff
@@ -74,6 +78,9 @@ export function PrivatePageLayout() {
               <AppShell className={css({ '& *': { colorPalette: isLocalEnv ? 'mode.local' : 'mode.cloud' } })}>
                 <Sidebar />
                 <ContentShell>
+                  {location.pathname !== '/get-started' && environment?.bridge?.url === BRIDGE_SYNC_SAMPLE_ENDPOINT && (
```
Comment (scopsy) [inline apps/web/src/components/layout/components/PrivatePageLayout.tsx]: Only visible when the synced url is equal to our sample endpoint

**Comment by djabarovgeorge** [comment]
Found this white like on the top, could it be related to this PR changes?
image

File: apps/web/src/components/layout/components/v2/SampleWorkflowsBanner.tsx
Comment (djabarovgeorge) [inline apps/web/src/components/layout/components/v2/SampleWorkflowsBanner.tsx:1]: Should we convert the styles to use panda styles instead here?

File: apps/web/src/pages/auth/components/QuestionnaireForm.tsx [hunk 4]
```diff
@@ -75,6 +65,18 @@ export function QuestionnaireForm() {
...
     await refetchEnvironments();
 
+    try {
+      await api.post(`/v1/bridge/sync`, {
+        bridgeUrl: BRIDGE_SYNC_SAMPLE_ENDPOINT,
+        source: 'sample-workspace',
+      });
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(e);
```
Comment (djabarovgeorge) [inline apps/web/src/pages/auth/components/QuestionnaireForm.tsx:75]: Do we need this log?

Comment (scopsy) [reply apps/web/src/pages/auth/components/QuestionnaireForm.tsx:75]: Yes, I think it makes sense to log it to console, otherwise will be silent

**Comment by scopsy** [comment]
@djabarovgeorge good catch on the top menu clipping, fixed it.

**Review by djabarovgeorge — APPROVED** [review]
Looks good to me, tested it locally workers as expected, please note the comments i left.
In addition i wonder if we should keep the sample workflow in case the user initialized his environment, but then either closed it. and then he or the product would like to play with Novu but would not be able to because they wont have running Bridge.



Everything seems to be working! I've tested it locally and it functions as intended. There are a few comments I've added for your reference.

On a separate note, I have a question. Should we keep the sample workflow in case a user sets up their environment, but then later closes it? This way, if he (or the product team members) want to use Novu again for testing, they wouldn't need a running Bridge because they already have the sample workflow.

**Pull request merged**